test/fake_modules/bad_es6/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "optimist": "~0.6.0",
```

Comment:
sorry, same here. and could you make the test case stronger? e.g., assert that, it is `optimist` not used.